### PR TITLE
VirtualBox Troubleshooting

### PR DESCRIPTION
**Pull request by tomjn**
VirtualBox  specific troubleshooting

### CODE REVIEW

**Comment by Mte90** [comment]
What is missing to move this one?

**Comment by tomjn** [comment]
@Mte90 removing the parts that moved to this new file, but importantly, nobody has reviewed what it says! I could have put a big list of swear words in the middle :) The actual document itself needs to be reviewed

**Comment by Mte90** [comment]
It is mainly for MacOS, I checked the part for windows and others and seems fine (last 2 paragraphs)

File: docs/en-US/troubleshooting/virtualbox.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+permalink: /docs/en-US/troubleshooting/virtualbox
+---
+
+## Updating to MacOS Big Sur or Newer
+
+You need to update to a newer version of VirtualBox. Download the latest version of VirtualBox and install it.
+
+## Apple Silicon
+
+VirtualBox does not support Apple Silicon, you will need to find an alternative provider. We currently do not have hardware to test this, and there are no recommended providers. We may provide docker as a provider in the future, but there is no timeframe on this. When we do, we will announce it, so if you don't see documentation for Docker, then it is not supported or usable. We do however want help with this!
```
Comment (evertiro) [inline docs/en-US/troubleshooting/virtualbox.md]: I'd argue this whole paragraph @tomjn

Comment (evertiro) [reply docs/en-US/troubleshooting/virtualbox.md]: Other than the first sentence.

**Review by evertiro — COMMENTED** [review]
One tiny point of contention. Otherwise, I read the whole thing and see no swear words. I didn't test every command, but functionally it looks good to me.

**Pull request merged**